src/devices/Pn532/README.md

```diff
@@ -0,0 +1,221 @@
+```
+
+Example pooling a 14443 type B card like a credit card:
+
+```csharp
+byte[] retData = null;
+while ((!Console.KeyAvailable))
+{
+	retData = pn532.AutoPoll(5, 300, new PollingType[] { PollingType.Passive106kbpsISO144443_4B });
+	if (retData != null)
+	{
+		if (retData.Length >= 3)
+			break;
+	}
+
+	// Give time to PN532 to process
+	Thread.Sleep(200);
+}
+
+if (retData == null)
+	return;
+
+//Check how many tags and the type
+Console.WriteLine($"Num tags: {retData[0]}, Type: {(PollingType)retData[1]}");
+var decrypted = pn532.TryDecodeData106kbpsTypeB(retData.AsSpan().Slice(3));
+```
+
+## Reading or writing to cards
+
+PN532 implement a ReadWrite function that allows to use a high level Mifare card class. This implementation abstract the reader which is used.
+
+Once detected and selected like in the previous example, this fully dump the content of a classical Mifare 1K card:
+
+```csharp
+if (decrypted != null)
+{
+	Console.WriteLine($"Tg: {decrypted.TargetNumber}, ATQA: {decrypted.Atqa} SAK: {decrypted.Sak}, NFCID: {BitConverter.ToString(decrypted.NfcId)}");
+	if (decrypted.Ats != null)
+		Console.WriteLine($", ATS: {BitConverter.ToString(decrypted.Ats)}");
+	
+	MifareCard mifareCard = new MifareCard(pn532, decrypted.TargetNumber) { BlockNumber = 0, Command = MifareCardCommand.AuthenticationA };
+	mifareCard.SetCapacity(decrypted.Atqa, decrypted.Sak);
+	mifareCard.SerialNumber = decrypted.NfcId;
+	mifareCard.KeyA = new byte[6] { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
+	mifareCard.KeyB = new byte[6] { 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF };/
+	for (byte block = 0; block < 64; block++)
+	{
+		mifareCard.BlockNumber = block;
+		mifareCard.Command = MifareCardCommand.AuthenticationB;
+		var ret = mifareCard.RunMifiCardCommand();
+		if (ret < 0)
+		{
+			// Try another one
+			mifareCard.Command = MifareCardCommand.AuthenticationA;
+			ret = mifareCard.RunMifiCardCommand();
+		}
+
+		if (ret >= 0)
+		{
+			mifareCard.BlockNumber = block;
+			mifareCard.Command = MifareCardCommand.Read16Bytes;
+			ret = mifareCard.RunMifiCardCommand();
+			if (ret >= 0)
+				Console.WriteLine($"Bloc: {block}, Data: {BitConverter.ToString(mifareCard.Data)}");
+			else
+			{
+				Console.WriteLine($"Error reading bloc: {block}, Data: {BitConverter.ToString(mifareCard.Data)}");
+			}
+
+			if (block % 4 == 3)
+			{
+				// Check what are the permissions
+				for (byte j = 3; j > 0; j--)
+				{
+					var access = mifareCard.BlockAccess((byte)(block - j), mifareCard.Data);
+					Console.WriteLine($"Bloc: {block - j}, Access: {access}");
+				}
+				var sector = mifareCard.SectorTailerAccess(block, mifareCard.Data);
+				Console.WriteLine($"Bloc: {block}, Access: {sector}");
+			}
+		}
+		else
+		{
+			Console.WriteLine($"Autentication error");
+		}
+	}
+```
+
+## PN532 as a target
+
+It's possible to change the PN532 mode to be seen as a target byt another reader. A phone with NFC for example. The bellow example shows how to transform the PN532 into a Credit Card:
```

Comment:
github is showing last character here in red. I suspect it's not `:` and instead some other unicode.